features/Dockefile.buildimage

```diff
@@ -1,20 +1,31 @@
 FROM fedora:31
+
 ENV GOPATH /go
 ENV GOBIN /go/bin
 ENV GOCACHE /go/.cache
-ENV PATH=$PATH:$GOPATH/bin
-ENV GOVERSION golang-1.13.5-1.fc31
+ENV GOVERSION 1.13.5
+ENV PATH=$PATH:/root/.gimme/versions/go"$GOVERSION".linux.amd64/bin:$GOBIN
 
 # rpms required for building and running test suites
 RUN dnf -y install \
-    $GOVERSION \
+    gcc \
+    git \
     make \
-    tar \
     gettext \
+    which \
```

Comment:
i think the pkg changes here don't apply.

Reply:
what do you mean by pkg changes?

Reply:
the addition/removal of the install packages. those changes don't seem relevant here.

Reply:
Hmm.. I can see how git, which, findutils and python2 are relevant. Even gcc. But does gimme really require skopeo?

Reply:
good catch, no (copy paste from other repo where it's needed indeed)

Reply:
fixed